tests/TestCase/Console/CommandCollectionTest.php

```diff
@@ -123,18 +122,6 @@ public function testAddInstance()
         $this->assertSame($command, $collection->get('routes'));
     }
 
-    /**
-     * Instances that are not shells should fail.
-     */
-    public function testAddInvalidInstance()
-    {
-        $this->expectException(\InvalidArgumentException::class);
-        $this->expectExceptionMessage('Cannot use \'stdClass\' for command \'routes\'. It is not a subclass of Cake\Console\Shell');
-        $collection = new CommandCollection();
-        $shell = new stdClass();
-        $collection->add('routes', $shell);
```

Comment:
Is this possible with commands?

Reply:
The 2nd argument now has union type `string|CommandInterface` so this test is no longer necessary.